src/static/containers/ZeroCard.jsx

```diff
@@ -0,0 +1,248 @@
+                    <Col span={5}>
+                        <Rate
+                            disabled
+                            defaultValue={waste_reduction_score}
+                            character={<Icon type="delete" />}
+                        />
+                        <Rate disabled defaultValue={cost_score} character="€" />
+                        <Rate disabled defaultValue={difficulty_score} character={<Icon type="tool" />} />
+                    </Col>
+                </Card>
+                <Modal
+                    destroyOnClose
+                    visible={visible}
+                    title={title}
+                    width={1000}
+                    onCancel={this.close}
+                    footer={<Button key="back" onClick={this.close}>Fermer</Button>}
+                >
+                    <Row>
+                        <Col span={19}>
+                            {description}
+                        </Col>
+                        <Col span={5}>
+                            <Rate
+                                disabled
+                                defaultValue={waste_reduction_score}
+                                character={<Icon type="delete" />}
+                            />
+                            <Rate disabled defaultValue={cost_score} character="€" />
+                            <Rate disabled defaultValue={difficulty_score} character={<Icon type="tool" />} />
+                        </Col>
+                        { (help_links) ? (
+                            <Col span={19}>
+                                <h2>Lien pour vous aider dans cet objectif</h2>
+                                {help_links}
+                            </Col>
+                        ) : null }
+                    </Row>
+                    { hasStats ? (
+                        <Row>
+                            <h1>Statistiques</h1>
+                            <ul>
+                                { waste_reduction_score
+                                    ? <li key="waste_reduction_score">
+                                        Réduction de déchets&nbsp;: {waste_reduction_score} kg/an
+                                      </li>
+                                    : null }
+                                { co2_reduction
+                                    ? <li key="co2_reduction">
+                                        Réduction d&apos;émissions de CO²&nbsp;: {co2_reduction} kg/an
+                                      </li>
+                                    : null }
+                                { water_use_reduction
+                                    ? <li key="water_use_reduction">
+                                        Réduction d&apos;eau consommée&nbsp;: {water_use_reduction} l/an
+                                      </li>
+                                    : null }
+                            </ul>
+                        </Row>
+                    ) : null }
```

Comment:
Maybe use a function to create that in order not to pollute your render func, but it can be kept like that if you prefer